--- Rakefile.ori	2020-06-30 11:14:07.520696281 +0200
+++ Rakefile		2020-06-30 11:15:43.449706719 +0200
@@ -252,8 +252,13 @@
     when /cachewebsite(?:=([a-z]+))?/
       value = Regexp.last_match(1)
       if releasing?
-        warn "Building a release ⇒ ignoring #{opt} build option"
-        ENV['TAILS_WEBSITE_CACHE'] = '0'
+        if value != "force"
+           warn "Building a release ⇒ ignoring #{opt} build option"
+           ENV['TAILS_WEBSITE_CACHE'] = '0'
+        else
+           warn "Building a release ⇒ applying forced #{opt} build option"
+           ENV['TAILS_WEBSITE_CACHE'] = '1'
+        end
       else
         value = 'yes' if value.nil?
         case value
